src/v2/components/TopBar/components/PrimarySearch/index.tsx

```diff
@@ -24,6 +25,7 @@ const Results = styled(Box)`
 interface PrimarySearchProps {
   scheme: 'DEFAULT' | 'GROUP'
   history: any
+  isErrorPage: boolean
```

Comment:
This component shouldn't be remotely concerned with whether or not we're in an error page.